### PR TITLE
Find command bug

### LINKED ISSUE #126: [PE-D] Find command results does not seem to match keyword
As seen above, typing 'find he' directly into the app after listing the qAndAs shows some results that do not match.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: bowei-yu/ped#1

### PR DESCRIPTION
**Pull request by yongmingyang**
Find find command bug with reference from https://stackoverflow.com/questions/36793397/check-if-string-contains-word-not-substring, found by Jian Ling.

close #126

### CODE REVIEW

File: src/main/java/seedu/medmoriser/model/qanda/QuestionContainsKeywordsPredicate.java [hunk 1]
```diff
@@ -16,7 +17,12 @@ public QuestionContainsKeywordsPredicate(List<String> keywords) {
     @Override
     public boolean test(QAndA qAndA) {
         return keywords.stream()
-                .anyMatch(keyword -> qAndA.getQuestion().question.toLowerCase().contains(keyword.toLowerCase()));
+                .anyMatch(keyword -> {
+                    System.out.println(keyword);
```
Comment (joshruien) [inline src/main/java/seedu/medmoriser/model/qanda/QuestionContainsKeywordsPredicate.java]: Extra println statement?

File: src/main/java/seedu/medmoriser/model/qanda/QuestionContainsKeywordsPredicate.java [hunk 2]
```diff
@@ -16,7 +17,12 @@ public QuestionContainsKeywordsPredicate(List<String> keywords) {
...
+                    System.out.println(qAndA.getQuestion().question.toLowerCase());
```
Comment (joshruien) [inline src/main/java/seedu/medmoriser/model/qanda/QuestionContainsKeywordsPredicate.java]: This too

**Review by joshruien — CHANGES_REQUESTED** [review]
Minor changes but looks good to me!

**Review by joshruien — APPROVED** [review]

**Review by jonfoocy — APPROVED** [review]
LGTM

**Pull request merged**